Source/Extensions/Blazorise.Components/ListView.razor.cs

```diff
@@ -124,6 +125,11 @@ protected string ListGroupStyleNames
         /// </summary>
         [Parameter] public RenderFragment ChildContent { get; set; }
 
+        /// <summary>
+        /// Specifies the content to be rendered inside each item of the <see cref="ListView{TItem}"/>.
+        /// </summary>
+        [Parameter] public RenderFragment<ItemContext<TItem>> ItemContent { get; set; }
```

Comment:
We have settled on the `*Template` suffix so better to stick with it. Rename it to `ItemTemplate`.

Reply:
Right, was just because there was already a `ChildContent`

Reply:
Well, I think it would confuse people. ChildContent and ItemContent seem like the same thing for an untrained eye.